wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -1136,6 +1136,33 @@ public synchronized double getAngle(IMUAxis axis) {
     return 0.0;
   }
 
+  /**
+   * Returns the Yaw axis angle in degrees (CCW positive).
+   *
+   * @return The Yaw axis angle in degrees (CCW positive).
+   */
+  public synchronized double getAngle() {
+    switch (m_yaw_axis) {
+      case kX:
+        if (m_simGyroAngleX != null) {
+          return m_simGyroAngleX.get();
+        }
+        return m_integ_angle_x;
+      case kY:
+        if (m_simGyroAngleY != null) {
+          return m_simGyroAngleY.get();
+        }
+        return m_integ_angle_y;
+      case kZ:
+        if (m_simGyroAngleZ != null) {
+          return m_simGyroAngleZ.get();
+        }
+        return m_integ_angle_z;
+      default:
+    }
+    return 0.0;
```

Comment:
```suggestion
    return getAngle(m_yaw_axis);
```

Reply:
I chose to not go with this as it has an extra switch case that can be avoided.